Tests/DatadogTests/Datadog/Core/FeaturesConfigurationTests.swift

```diff
@@ -324,6 +324,59 @@ class FeaturesConfigurationTests: XCTestCase {
         )
     }
 
+    // MARK: - Crash Reporting Configuration Tests
+
+    func testWhenCrashReportingIsDisabled() throws {
+        XCTAssertNil(
+            try FeaturesConfiguration(configuration: .mockWith(crashReportingPlugin: nil), appContext: .mockAny()).crashReporting,
+            "Feature configuration should not be available if the feature is disabled"
+        )
+    }
+
+    func testWhenRUMorLoggingFeaturesAreEnabled_thenCrashReportingFeatureCanBeEnabled() throws {
+        let random = [true, true, false].shuffled() // get at least one `true` on first two positions
```

Comment:
This is a more of a general point (maybe addressable in RUMM-896). I'm wondering what's the impact of randomizing test inputs and if it could lead to flakiness by randomly uncovering a regression.

Reply:
Definitely something we can discuss in `RUMM-896`! I find this sort of [property testing](https://en.wikipedia.org/wiki/Property_testing) as a practical approach to covering multiple inputs leading to a single result. By definition, this should not lead to flakiness as this randomization should be build very consiously.